projects/wave-core-new/src/lib/operators/datatype.model.ts

```diff
@@ -30,7 +30,7 @@ class Byte extends DataType {
     }
 
     getCode(): string {
-        return 'Byte';
+        return 'U8';
     }
 
     getMin(): number {
```

Comment:
Maybe we should refactor it into raster data types and vector data types…
…we can't use strings in rasters and we can't have u16 in vector data.

…and if we do it, we could change the methods that output the same string into readonly fields.

Reply:
shall I do it now, or do we do it, when we add the first vector operator?

Reply:
You can decide, but if you say we do it later, you should add it as a note to https://trello.com/c/tPOoZbu5/131-point-in-polygon-operator-anpassen

Reply:
I added a note